<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthForm</class>
 <widget class="QWidget" name="AuthForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0#3c1053, stop:1 #ad5389);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>405</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>60</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>60</number>
    </property>
    <item>
     <widget class="QLabel" name="title_label">
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 30px;
background: rgba(255,255,255, 0);</string>
      </property>
      <property name="text">
       <string>Авторизация</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_login">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(255, 255, 255, 0.5);
height: 29px;
border-radius: 10px;
border: 0;
padding: 10px;
color: white;
font-size: 24px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_password">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(255, 255, 255, 0.5);
height: 29px;
border-radius: 10px;
border: 0;
padding: 10px;
color: white;
font-size: 24px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="login_btn">
      <property name="styleSheet">
       <string notr="true">border: 0;
border-radius: 10px;
font-size: 24px;
padding: 10px;
background: #179bf3;
color: white;
</string>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="singup_btn">
      <property name="styleSheet">
       <string notr="true">border: 0;
border-radius: 10px;
font-size: 24px;
padding: 10px;
background: rgb(12, 189, 18);
color: white;
</string>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="error_label">
      <property name="styleSheet">
       <string notr="true">color: red;
border: 0;
background: rgba(255,255,255, 0%)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
